package.json

```diff
@@ -29,7 +29,7 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test",
+    "test": "react-scripts test jest",
```

Comment:
I don't think we need this `jest` bit. React-scripts automatically uses jest. Pls remove

Reply:
removed